src/sinks/websocket/sink.rs

```diff
@@ -236,6 +236,16 @@ impl WebSocketSink {
         Ok(())
     }
 
+    fn should_encode_as_binary(&self) -> bool {
```

Comment:
ok final thing (which is unfortunately blocking) - clippy is going to complain about this function not being `const`. 

In general it's good practice to run `make check-clippy` before you push to your remote branch.